text_localization_environment/TextLocEnv.py

```diff
@@ -117,20 +140,36 @@ def to_four_corners_array(two_bbox):
 
         return np.array([four_bbox, four_bbox, four_bbox])
 
-    def create_ior_mark(self):
+    @staticmethod
+    def to_standard_box(bbox):
```

Comment:
I was wondering before: why are the agent bounding box and the true bounding boxes not the same format?
Wouldn't it be better to standardize the initial representation instead of converting them later?

Reply:
Agreed. Didn't change the original representation because I wasn't very familiar with the project and unsure if it's used somewhere else (e.g. agent repo). Feel free to refactor & create a PR!